specifyweb/frontend/js_src/lib/components/SchemaViewer/helpers.tsx

```diff
@@ -4,6 +4,7 @@ import { useNavigate, useParams } from 'react-router-dom';
 import type { IR } from '../../utils/types';
 
 export type SchemaViewerValue =
+  | boolean
```

Comment:
hmm, I don't think the boolean type should be included here (if I remember correctly)
instead, you should call `formatBoolean(someBooleanValue)` in your code to convert boolean to a localized value

Reply:
Ah, this boolean is not shown to the user (at least in my use case). It was done because I needed a way to flag a row to have some special classNames (highlight a bit red on error) Actually, I think I can avoid this, I can define a callback that flags it during runtime (and takes in cell and rows)